pages/documentation/getting-started/sample-contract-language.md

```diff
@@ -0,0 +1,223 @@
+---
+permalink: /documentation/sample-contract/
+layout: styleguide
+title: Sample contract language for 21st Century IDEA
+category: How to use USWDS
+lead: How to document requirements related to website modernization in contracts
+redirect_from:
+  - /contract-language
+  - /contracts
+---
+
+The [21st Century Integrated Digital Experience Act](https://digital.gov/resources/21st-century-integrated-digital-experience-act/)
+(21st Century IDEA) requires all executive branch agencies to:
+
+- modernize their websites
+- digitize services and forms
+- accelerate use of e-signatures
+- improve customer experience
+- standardize and transition to centralized shared services
+
+It also requires agencies to comply with [website standards](https://designsystem.digital.gov/website-standards/) to improve federal websites and digital services.
+
+Use the approaches below as a starting point for any solicitations with requirements related to websites and digital services.
+
+## Before you get started
+
+21st Century IDEA codifies high-level requirements for federal websites and digital services. Additionally, many agencies have more specific requirements in their digital strategy and other guidelines.
+
+Consult with your agency’s Chief Information Officer and [Federal Web Council representative](https://digital.gov/resources/federal-web-council/#current-council-members) regarding agency requirements for websites and digital services, including 21st Century IDEA.
+
+Then work with your contracting officer (CO) to determine your acquisition strategy, including customizing the draft language below to meet your needs and requirements.
+
+## Draft contract language
+
+### New solicitations
+
+In new solicitations, you can include requirements for 21st Century IDEA in the performance work statement.
+
+#### Draft language for new solicitations
```

Comment:
@afeijoo I've been formatting this page and I have a few instances where I'm not sure if the heading is part of the contract language or meant to introduce it. Can you look at each of these (this is the first one) and let me know if the heading should be in the blockquote or outside of it.

Reply:
[Preview link for reference](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/as-add-contract-lang/documentation/sample-contract/) →